treeherder/changelog/migrations/0001_squashed_0003_auto_20200207_0902.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 3.0.2 on 2020-02-07 09:16
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('changelog', '0001_initial'), ('changelog', '0002_auto_20200207_0809'), ('changelog', '0003_auto_20200207_0902')]
+
```

Comment:
Please rename any migration files so they are sequential and descriptive, such as in `treeherder/model/migrations`.

Reply:
Also, some of these you can squash into one. Probably destroying the container/mysql instance locally and starting over will ensure the migrations don't get out of sync locally.